sdk/storage/azure-storage-blob/azure/storage/blob/_shared/base_client.py

```diff
@@ -284,10 +282,17 @@ def _batch_send(
             enforce_https=False
         )
 
+        Pipeline._prepare_multipart_mixed_request(request) # pylint: disable=protected-access
```

Comment:
I can create a` _prepare_multipart_mixed_request` in azure-storage-blob package later, while we still cannot avoid this hacky way to deserialize the response.
```
response = pipeline_response.http_response
request.multipart_mixed_info = temp
```